docs/installation.rst

```diff
@@ -49,6 +49,8 @@ If you want to enable haystack support, in addition to the above:
 * add ``aldryn_search`` to ``INSTALLED_APPS``
 * configure haystack according to `aldryn-search docs <https://github.com/aldryn/aldryn-search#usage>`_
   and `haystack docs <http://django-haystack.readthedocs.io/en/stable/>`_.
+* if not using ``aldryn_search``, you can define your own index by skipping ``aldryn_search`` installation and writing
```

Comment:
Maybe replace `your index` by `your own search_indexes.py` in order to give potential users a hint of what to add ?

Reply:
thanks for the suggestion, I reworded the documentation according to this comment